### PR TITLE
[WIP] created user api using sequalize and ORM structure

### PR DESCRIPTION
**Pull request by DeveloperYogesh07**

### CODE REVIEW

File: code.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+const express = require("express");
```
Comment (rakeshsawan) [inline code.js]: why the file is named as code.js? we need to have a better name for this.

File: src/controller/controller.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
+import { Request, Response, NextFunction } from "express";
+import { genSaltSync, hashSync, compareSync } from "bcryptjs";
+import UserModel from "../models/users";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+
+interface UserInterface {
```
Comment (rakeshsawan) [inline src/controller/controller.ts]: we can create a class and implement all necessary method. also the controller.ts should be reame to userController.ts

File: src/controller/controller.ts [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  id?: number;
+  name: string;
+  email: string;
+  password: string;
+  role: string;
+}
+
+const getAllUsers = async (req: Request, res: Response, next: NextFunction) => {
+  try {
```
Comment (rakeshsawan) [inline src/controller/controller.ts]: let's avoid passing next function. instead use async await pattern.

File: code.js (hunk 1 above)
Comment (DeveloperYogesh07) [reply code.js]: It's not required i will remove it.

**Comment by DeveloperYogesh07** [comment]
i am closing this PR because I have moved the changes in useraApi_v2.

**Pull request closed without merging**